ansible/playbooks/roles/grafana/tasks/upgrade/main.yml

```diff
@@ -71,3 +71,6 @@
       file:
         path: "{{ upgrade_defaults.state_file_path }}"
         state: absent
+
+- name: Run password change
+  include_tasks: password_change.yml
```

Comment:
Is it necessary here? I don't see such file in `upgrade`.

Reply:
I wanted it to be able to change password during the upgrade too. It uses the same file as apply does (mode independent).